test/integration/controllers/auth.spec.js

```diff
@@ -132,12 +132,20 @@ describe('Auth controller', () => {
   })
 
   describe('Login endpoint', () => {
-    it('should login a user', async () => {
+    let confirmToken
+    beforeEach(async () => {
       const findConfirmTokenResponse = await tokenService.findTokensWithUsersByParams({
         user: signupResponse.body.userId
       })
-      const confirmToken = findConfirmTokenResponse[0].confirmToken
+      confirmToken = findConfirmTokenResponse[0].confirmToken
+
+      Token.findOne = jest
+        .fn()
+        .mockResolvedValue({ ...{ confirmToken }, save: jest.fn().mockResolvedValue(confirmToken) })
```

Comment:
i think here is no need to spread this object with confirmToken, just add it as a property same to field "save"
the same would be good to refactor in tests below

Reply:
Fixed.